src/nnet3/nnet-component-itf.h

```diff
@@ -471,6 +471,9 @@ class UpdatableComponent: public Component {
   /// learning_rate_factor_.
   virtual void SetAsGradient() { learning_rate_ = 1.0; is_gradient_ = true; }
 
+  /// freezes/unfreezes NaturalGradient updates
```

Comment:
add: `, if applicable (to be overriden by components that use Natural Gradient).`.
Also, please make sure that all components that use natural gradient actually do
override this, including ones that were added since you started this work.  
Does TimeHeightConvolutionCompoent override this?  And LstmNonlinearityComponent,
if it uses natural gradient? (I don't recall).
If not, it might impact @GaofengCheng's work.